Happy-Hour/src/main/resources/templates/owner-registration.html

```diff
@@ -4,44 +4,40 @@
 </head>
 <title th:text="${title}"></title>
 
-<body class="container bg" style="font-family: Comic Sans MS, Comic Sans, cursive; font-weight: bold; color: white;">
+<body class="container body_text">
+    <div class="bg">
+        <div th:replace="fragments :: header"></div>
 
-    <div class="page-header text-center my-5">    <!--This header should be fragmented-->
-        <h1>Happy Hour Locator</h1>
-    </div>
-    <div class="navbar-header d-flex justify-content-flex-start text-center mb-4">  <!--This nav bar should be fragmented-->
-        <a th:href="@{/index}" class="navbar-brand m-0 px-1 border">Home</a>
-        <p th:text="${title}" class="navbar-brand m-0 w-100 border"></p>
-    </div>
+        <div th:replace="fragments :: navbar"></div>
 
-    <div class="container h-100">
-        <div class="d-flex justify-content-center">
-            <div class="row h-100 justify-content-center align-items-center">
-                <form method="post">
-                    <div class="form-group">
-                        <label>Username
-                            <input class="form-control" th:field="${registerFormDTO.username}"/>
-                        </label>
-                        <p class="error" th:errors="${registerFormDTO.username}"></p>
-                    </div>
-                    <div class="form-group">
-                        <label>Password
-                            <input class="form-control" th:field="${registerFormDTO.password}" type="password"/>
-                        </label>
-                        <p class="error" th:errors="${registerFormDTO.password}"></p>
-                    </div>
-                    <div class="form-group">
-                        <label>Verify Password
-                            <input class="form-control" th:field="${registerFormDTO.verifyPassword}" type="password"/>
-                        </label>
-                    </div>
+        <div class="container h-100">
+            <div class="d-flex justify-content-center">
+                <div class="row h-100 justify-content-center align-items-center">
```

Comment:
and remove this. Making these changes should recenter the registration form and place it below the header and navbar

Reply:
removed div element.